app/javascript/dashboard/mixins/uiSettings.js

```diff
@@ -1,10 +1,28 @@
 import { mapGetters } from 'vuex';
-
+export const DEFAULT_CONVERSATION_SIDEBAR_ITEMS_ORDER = [
+  { name: 'conversation_info' },
+  { name: 'contact_attributes' },
+  { name: 'previous_conversation' },
+  { name: 'conversation_actions' },
```

Comment:
@muhsin-k Can you please change the default order to 

conversation_actions,
conversation_info,
contact_attributes,
previous_conversation